docs/open-source-tizen/developing/building-all.md

```diff
@@ -330,19 +334,19 @@ $ cd <Tizen_Project>
 $ gbs build -A i586 --threads=4 --clean-once
 ```
 
-> **Note**
+> **Note**  
 > Since the GBS configuration file (`.gbs.conf`) and build configuration file (`build.conf`) are also included in `scm/manifests`, they are automatically downloaded in the following paths after the `repo sync`:
 >
 > - `.gbs.conf`: `~/<Tizen_Project>/.gbs.conf`
 > - `build.conf`: `~/<Tizen_Project>/scm/meta/build-config/<Tizen version>/<profile>/<repository>_build.conf`
 
 ## GBS Build Concepts <a name="gbs-build-concepts"></a>
 
-The following build concepts help you transparently understand the full build commands:
+The following build concepts help you understand the full build commands:
 
 - **Dependency cycles**
-  - In Tizen versions higher than Tizen 2.4, packages with dependency cycles are only included in the Tizen:[<Tizen version>]:Base OBS project. And building packages with dependency cycles are not supported by GBS, so it is impossible to build all packages locally with GBS for Tizen:[<Tizen version>]:Base. Therefore, a remote repo which contains RPMs built by OBS for Tizen:[<Tizen version>]:Base needs to be added in the `.gbs.conf` file.
-  - In Tizen versions lower than Tizen 2.3, there is only one OBS project and packages with dependency cycles are included in it. To continue the build by breaking the dependency cycles for Tizen versions lower than Tizen 2.3, see [Excluding Specific Packages](building.md#id3).
+  - In Tizen versions higher than Tizen 2.4, packages with dependency cycles are only included in the `Tizen:[<Tizen version>]:Base` OBS project. Since building packages with dependency cycles is not supported by GBS, it is impossible to build all packages locally with GBS for `Tizen:[<Tizen version>]:Base`. Therefore, a remote repo which contains RPMs built by OBS for `Tizen:[<Tizen version>]:Base` needs to be added in the `.gbs.conf` file.
+  - In Tizen versions lower than Tizen 2.3, there is only 1 OBS project and packages with dependency cycles are included in it. To continue the build by breaking the dependency cycles for Tizen versions lower than Tizen 2.3, see [Excluding Specific Packages](building.md#id3).
 - **Accelerator packages**
```

Comment:
The accelerator package information covers Tizen 3.0 and higher, and Tizen 2.3 and lower. How does this work in Tizen 2.4?

Reply:
OK, this will be fixed